src/lib/Helper/ContentHelper.php

```diff
@@ -156,11 +156,11 @@ private function getQuery(int $contentTypeId, array $options): Query
             new Criterion\ContentTypeId($contentTypeId),
         ];
 
-        if ($options['path']) {
+        if (isset($options['path'])) {
             $criteria[] = new Criterion\Subtree($options['path']);
         }
 
-        if (!$options['hidden']) {
+        if (!isset($options['hidden'])) {
```

Comment:
So, if `$options['hidden']` is `true`, then this criterion (`Visibility::VISIBLE`) would be applied?

That seems incorrect.

```suggestion
        if ($options['hidden'] ?? true) {
```

Reply:
This check should be removed because parameter hidden is no longer available (dropped here: https://github.com/ezsystems/ezrecommendation-client/pull/67/files#diff-38d38da1d4111bc75c7cf6a74f3b4eb4a574beabf92e4e731065bec071a39c7cL70). 

Now dedicated QueryType is used to build query: https://github.com/ezsystems/ezrecommendation-client/blob/master/src/lib/QueryType/ContentDataSourceQueryType.php#L52